### DISCUSSION TITLE
Query by country

### DISCUSSION THREAD

**Discussion by nbeecher** [OvertureMaps/data#97] [Discussion]
How do I query admin data by country easily?  Looks like only adminLevel 2 has the iso country code.  

Example: I want to know how many admin levels the US currently has.

Thanks!

**Comment by mtravis** [comment]
Are you using the latest data release. There isn't a country code at the other admin levels as of yet but there is a context id column that links the 2 to 4, etc.

What are you using to query the data?

**Reply by nbeecher** [reply]
Thanks Matt.  I'm currently using DuckDB to query the data.  I will try leveraging the context id but was hoping there was an easier way to do this ha.  Thanks again for your help.
